api/features/models.py

```diff
@@ -989,6 +992,41 @@ def copy_identity_feature_states(
             # Save changes to target feature_state
             target_feature_state.save()
 
+    @hook(AFTER_SAVE)
+    def create_github_comment(self) -> None:
+        from integrations.github.github import GithubData, generate_data
+        from integrations.github.tasks import (
+            call_github_app_webhook_for_feature_state,
+        )
+        from webhooks.webhooks import WebhookEventType
+
+        if (
+            not self.identity_id
+            and not self.feature_segment
+            and self.feature.external_resources.exists()
+            and self.environment.project.github_project.exists()
+            and self.environment.project.organisation.github_config.exists()
+        ):
+            github_configuration = (
+                Organisation.objects.prefetch_related("github_config")
+                .get(id=self.environment.project.organisation_id)
+                .github_config.first()
+            )
```

Comment:
Err... what? Why not just: 

```suggestion
            github_configuration = (
                GithubConfig.objects.get(organisation_id=self.environment.project.organisation_id)
            )
```

Reply:
Long story, but basically I had issues with the `soft-delete` model and the `one-to-one` relationship and other things, but that complex query is not needed. Removed